.github/workflows/pr-build.yml

```diff
@@ -76,23 +84,37 @@ jobs:
         
       - name: Generate files with Builder
         run: dart run build_runner build --delete-conflicting-outputs 
+
+      - name: Analyze Dart code
+        uses: zgosalvez/github-actions-analyze-dart@v3
         
       - name: Build with Flutter
         continue-on-error: true
         id: flutter-build
         env: 
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          flutter build apk --${{ inputs.app-flavour }};
+          flutter build apk --${{ inputs.app-flavor }} ${{ inputs.flutter-build-arguments }}
           
       - name: Prepare to comment
         run: |
           if [[ "${{ steps.flutter-build.outcome }}" == "success" ]]; then
-            echo "MESSAGE=✅ ReVanced Manager ${{ env.COMMIT_HASH }} build succeeded." >> $GITHUB_ENV
+            echo "STATUS_MESSAGE=✅ ReVanced Manager ${{ env.COMMIT_HASH }} build succeeded." >> $GITHUB_ENV
           else
-            echo "MESSAGE=🚫 ReVanced Manager ${{ env.COMMIT_HASH }} build failed." >> $GITHUB_ENV
+            echo "STATUS_MESSAGE=🚫 ReVanced Manager ${{ env.COMMIT_HASH }} build failed." >> $GITHUB_ENV
+          fi
+          if [ -n "{{ inputs.flutter-build-arguments }}" ]; then
+            echo "COMPILE_ARGUMENT_MESSAGE=- Compile arguments: ${{ inputs.flutter-build-arguments }}" >> $GITHUB_ENV
           fi
           
+      - name: Upload build
+        uses: actions/upload-artifact@v3
+        with:
+          name: revanced-manager-(${{ env.COMMIT_HASH }}-${{ inputs.pr-number }}-${{ inputs.app-flavor }})-${{ inputs.flutter-branch }}
+          path: | 
+            build/app/outputs/flutter-apk/app-${{ inputs.app-flavor }}.apk
+            build/app/outputs/flutter-apk/app-${{ inputs.app-flavor }}.apk.sha1
+          
       - name: "Comment to Pull Request #${{ inputs.pr-number }}"
```

Comment:
Can you make this step comment as @revanced-bot?

Reply:
> Can you make this step comment as @revanced-bot?

This was mentioned in the initial pull request for ci(build), this suggestion depends on the repository manager.

Reply:
The repository manager being who? @oSumAtrIX can you look into this?